apps/userdashboard/templates/a4_candy_userdashboard/base_userdashboard.html

```diff
@@ -91,7 +91,7 @@ <h3>{% trans 'Organisations' %}</h3>
                           {% if organisation.logo %}
                             <img src="{% thumbnail organisation.logo '0x40' %}" alt="{{ orgnaisation.name }}" />
                           {% else %}
-                            <i class="fas fa-home fa--lg pt-1" aria-label="{{ orgnaisation.name }} {% trans 'Home' %}"></i>
+                            <i class="fas fa-home font--lg pt-1" aria-label="{{ orgnaisation.name }} {% trans 'Home' %}"></i>
```

Comment:
Hihi, orgnaisation.name might not work. Ups, also in the image below. I will still merge as this wasn't done here and add another issue. :)